src/operators/timeoutWith.ts

```diff
@@ -69,31 +68,27 @@ class TimeoutWithSubscriber<T, R> extends OuterSubscriber<T, R> {
   }
 
   _next(value: T) {
-    if (!this.timedOut) {
```

Comment:
Noticed I removed `timedOut`. It's not necessary because the check for `isUnsubscribed` that happens in the superclass `Subscriber` already guards against these cases.